samples/msal-node-silent-flow/data/cache.json

```diff
@@ -0,0 +1,110 @@
+{
+    "Account": {
+        "uid.utid-login.microsoftonline.com-microsoft": {
+            "username": "John Doe",
+            "local_account_id": "object1234",
+            "realm": "microsoft",
+            "environment": "login.microsoftonline.com",
+            "home_account_id": "uid.utid",
+            "authority_type": "MSSTS",
+            "client_info": "base64encodedjson"
+        },
+        "fb3680ec-7b98-4e1c-8434-5be16c5e78fd.90b8faa8-cc95-460e-a618-ee770bee1759-login.windows.net-90b8faa8-cc95-460e-a618-ee770bee1759": {
+            "home_account_id": "fb3680ec-7b98-4e1c-8434-5be16c5e78fd.90b8faa8-cc95-460e-a618-ee770bee1759",
+            "environment": "login.windows.net",
+            "realm": "90b8faa8-cc95-460e-a618-ee770bee1759",
+            "local_account_id": "fb3680ec-7b98-4e1c-8434-5be16c5e78fd",
+            "username": "user@sgonz.onmicrosoft.com",
+            "authority_type": "MSSTS",
+            "name": "Test User",
+            "client_info": "eyJ1aWQiOiJmYjM2ODBlYy03Yjk4LTRlMWMtODQzNC01YmUxNmM1ZTc4ZmQiLCJ1dGlkIjoiOTBiOGZhYTgtY2M5NS00NjBlLWE2MTgtZWU3NzBiZWUxNzU5In0"
+        }
+    },
+    "AccessToken": {
+        "uid.utid-login.microsoftonline.com-accesstoken-mock_client_id-microsoft-scope1 scope2 scope3": {
+            "environment": "login.microsoftonline.com",
+            "credential_type": "AccessToken",
+            "secret": "an access token",
+            "realm": "microsoft",
+            "target": "scope1 scope2 scope3",
+            "client_id": "mock_client_id",
+            "cached_at": "1000",
+            "home_account_id": "uid.utid",
+            "extended_expires_on": "4600",
+            "expires_on": "4600"
+        },
+        "uid.utid-login.microsoftonline.com-accesstoken-mock_client_id-microsoft-scope4 scope5": {
+            "environment": "login.microsoftonline.com",
+            "credential_type": "AccessToken",
+            "secret": "an access token",
+            "realm": "microsoft",
+            "target": "scope4 scope5",
+            "client_id": "mock_client_id",
+            "cached_at": "1000",
+            "home_account_id": "uid.utid",
+            "extended_expires_on": "4600",
+            "expires_on": "4600"
+        },
+        "fb3680ec-7b98-4e1c-8434-5be16c5e78fd.90b8faa8-cc95-460e-a618-ee770bee1759-login.windows.net-accesstoken-99cab759-2aab-420b-91d8-5e3d8d4f063b-90b8faa8-cc95-460e-a618-ee770bee1759-openid profile user.read email": {
```

Comment:
We shouldn't be committing tokens to samples. This is similar to committing passwords.